### DISCUSSION TITLE
When will an integration with Angular example be ready?

### DISCUSSION THREAD

**Discussion by playground** [WalletConnect/walletconnect-monorepo#1889] [Developer Support]
Is there a timeframe for the Angular example to be released?

**Comment by daofabdotcom** [comment]
bump

**Comment by devsktlabs** [comment]
bump

**Comment by glitch-txs** [comment]
Hi, someone from the community built this example for Angular https://github.com/csurbier/walletconnectv2-angular/tree/main

In general if you're facing a typescript issue make sure to add `"SkipLibCheck": true` in the tsconfig.json file.
